register.go

```diff
@@ -77,13 +81,13 @@ func (s *Snitch) register(looper director.Looper) {
 				return nil
 			}
 
-			if errors.Is(err, io.EOF) {
+			if errors.Is(err, io.EOF) || strings.Contains(err.Error(), "reading from server: EOF") {
```

Comment:
This is the case I told you about - sometimes `errors.Is()` won't catch the EOF and you have to do a `string.Contains()`.